libs/resources/RespCommandsInfo.json

```diff
@@ -1546,6 +1546,31 @@
       }
     ]
   },
+  {
+    "Command": "HCOLLECT",
```

Comment:
Add this new command to https://microsoft.github.io/garnet/docs/commands/garnet-specific-commands

Reply:
Done, should I remove it from https://microsoft.github.io/garnet/docs/commands/data-structures?

Reply:
It's fine to have it here, and point to Garnet specific.